app/lib/shared/handlers.dart

```diff
@@ -24,7 +24,7 @@ const staticLongCache = const Duration(hours: 24);
 shelf.Response redirectResponse(String url) => new shelf.Response.seeOther(url);
 
 shelf.Response redirectToSearch(String query) {
-  final uri = new Uri(path: '/packages', queryParameters: {'q': query});
+  final uri = searchUrl(q: query);
   return redirectResponse(uri.toString());
```

Comment:
`uri` is now a `String`, so we don't really need to call `toString()` on it. Doesn't hurt, though.

Reply:
Starting which SDK? I don't see it with dev-46.

Reply:
Sorry, I meant the `uri` variable. `searchUrl()` returns a `String`.